src/bun.js/bindings/generated_perf_trace_events.h

```diff
@@ -1,61 +1,60 @@
+                                        macro(Bundler.Worker.create, 9)                                                                                                                                                                                                                 \
+                                            macro(Bundler.WrapDependencies, 10)                                                                                                                                                                                                         \
+                                                macro(Bundler.breakOutputIntoPieces, 11)                                                                                                                                                                                                \
+                                                    macro(Bundler.cloneAST, 12)                                                                                                                                                                                                         \
+                                                        macro(Bundler.computeChunks, 13)                                                                                                                                                                                                \
+                                                            macro(Bundler.findAllImportedPartsInJSOrder, 14)                                                                                                                                                                            \
+                                                                macro(Bundler.findReachableFiles, 15)                                                                                                                                                                                   \
+                                                                    macro(Bundler.generateChunksInParallel, 16)                                                                                                                                                                         \
+                                                                        macro(Bundler.generateCodeForFileInChunkCss, 17)                                                                                                                                                                \
+                                                                            macro(Bundler.generateCodeForFileInChunkJS, 18)                                                                                                                                                             \
```

Comment:
Is there a magic comment to skip clang-format for generated code?

The original formatting is much easier to read compared to what was produced by clang-format.